packages/storybook/src/nimble/table-column/mapping/table-column-mapping.mdx

```diff
@@ -25,35 +33,51 @@ The <Tag name={tableColumnMappingTag}/> renders specific number, boolean, or str
 <Controls of={tableColumnMappingStories.mappingColumn} />
 <ComponentApisLink />
 
-### Icon Mapping
+### Icon mapping
 
 <Description of={iconMappingStories} />
 <Controls of={iconMappingStories.iconMapping} />
 
-### Spinner Mapping
+### Spinner mapping
 
 <Description of={spinnerMappingStories} />
 <Controls of={spinnerMappingStories.spinnerMapping} />
 
-### Text Mapping
+### Text mapping
 
 <Description of={textMappingStories} />
 <Controls of={textMappingStories.textMapping} />
 
-### Empty Mapping
+### Empty mapping
 
 <Description of={emptyMappingStories} />
 <Controls of={emptyMappingStories.emptyMapping} />
 
 ## Usage
 
-### Best Practices
+### Best practices
+
+-   Provide a mapping for every expected record value. Because grouping is
+    performed on the record value, non-mapped record values can result in
+    multiple groups without group labels.
+    -   To improve grouping behavior of a value that shouldn't be rendered in
+        table cells, use a <Tag name={mappingEmptyTag}/> element to provide text
+        for that value's group row.
+-   Avoid having multiple values that map to the same text because grouping and
+    sorting is done on the record value rather than the mapped text.
+
+#### Choosing a mapping style
 
--   Provide a mapping for every expected record value. Because grouping is performed on the record value, non-mapped record values can result in multiple groups without group labels.
-    -   To improve grouping behavior of a value that shouldn't be rendered in table cells, use a <Tag name={mappingEmptyTag}/> element to provide text for that value's group row.
--   Avoid having multiple values that map to the same text because grouping and sorting is done on the record value rather than the mapped text.
+| Best practice                                                                                                                                                                                  | Example                                      |
```

Comment:
Should we include the name of the API to configure this state in the guidance somewhere (`text-hidden`)? It could get pretty wordy but I think it could be helpful. For example: "Icon and text" -> "Icon and text (`text-hidden=false`)" and "Icon-only" -> "Icon-only (`text-hidden=true`)".

An alternate idea would be new column(s) that display the mapping configuration (and, if you think it's useful, the column header configuration). But that might use a lot of space.

Reply:
I tried to find a succinct way to add this to the table, but struggled in part because it's [not accurate](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML) to document `text-hidden=false` as equivalent to not adding the `text-hidden` attribute, correct? 

I'd argue that the attributes are named such that everyone will figure out the correct config without the docs spelling it out.